templates/internet-of-things/smart-displays.html

```diff
@@ -7,377 +7,541 @@
-                    alt="Ubuntu core",
-                    width="156",
-                    height="36",
-                    hi_def=True,
-                    loading="lazy",
-                    attrs={"class": "p-card__thumbnail"}) | safe
-          }}
-        </div>
-        <p class="p-heading--4">An operating system designed for the edge</p>
-        <div class="p-card__content">
-          <p>
-            With a small footprint and serious power, <a href="/core">Ubuntu Core</a> reduces hardware costs while still running 4k video. As a low-cost, open source solution, it builds from its heritage on servers to bring a secure and stable platform, designed for 24/7 availability, and resistant to screen takeovers and other attacks.
-          </p>
-          <p>
-            Ubuntu Core brings all the security features and device management solutions you need to deploy, provision, and maintain your device.
-          </p>
-        </div>
-        <p>
-          <a href="/core">Learn more about Ubuntu Core&nbsp;&rsaquo;</a>
-        </p>
-      </div>
-      <div class="p-card col-6">
-        <div class="u-sv1">
-          {{ image(url="https://assets.ubuntu.com/v1/cac4e35c-Ubuntu+Frame-logo-RGB-2022.png",
-                    alt="Ubuntu frame",
-                    width="172",
-                    height="36",
-                    hi_def=True,
-                    loading="lazy",
-                    attrs={"class": "p-card__thumbnail"}) | safe
-          }}
+      <div class="col-start-large-4 col-9 col-medium-6">
+        <div class="p-section--shallow">
+          <div class="row--25-75">
+            <hr class="p-rule--muted" />
+            <div class="col u-hide--small">
+              {{ image(url="https://assets.ubuntu.com/v1/d68b3378-ubuntu-frame-final.png",
+                        alt="",
+                        width="852",
+                        height="172",
+                        hi_def=True,
+                        loading="lazy") | safe
+              }}
+            </div>
+            <div class="col">
+              <div class="p-section--shallow">
+                <p>
+                  With a small footprint and serious power, <a href="/core">Ubuntu Core</a> reduces hardware costs while still being able to run 4k video.
+                  As a low-cost, open source solution, it builds on top of its server heritage to bring a secure and stable platform, designed for 24/7 availability,
+                  and resistant to screen takeovers and other attacks.
+                </p>
+                <p>
+                  Ubuntu Core brings all the security features and device management solutions you need to deploy, provision, and maintain your device.
+                </p>
+              </div>
+              <div class="p-cta-block">
+                <a href="/core">Learn more about Ubuntu Core&nbsp;&rsaquo;</a>
+              </div>
+            </div>
+          </div>
```

Comment:
Wondering if it was possible to use tiered list pattern for this section

Reply:
Thanks for the suggestion! The tiered list pattern is using 50/50 split on medium and small screens, but this section is using 25/75 so it's slightly different. I will leave it as is for now